pkg/quota/evaluator/core/pods.go

```diff
@@ -144,10 +157,15 @@ func PodUsageFunc(object runtime.Object) api.ResourceList {
 	// when we have pod level cgroups, we can just read pod level requests/limits
 	requests := api.ResourceList{}
 	limits := api.ResourceList{}
+
 	for i := range pod.Spec.Containers {
 		requests = quota.Add(requests, pod.Spec.Containers[i].Resources.Requests)
 		limits = quota.Add(limits, pod.Spec.Containers[i].Resources.Limits)
 	}
+	for i := range pod.Spec.InitContainers {
+		requests = quota.Max(requests, pod.Spec.InitContainers[i].Resources.Requests)
```

Comment:
Am i reading this right @smarterclayton...We add up say ResourceRequestsCPU for all the containers and compare with each init container? (instead of adding all the initcontainers ?)

Thanks,
Dims